gobblin-modules/gobblin-http/src/main/java/org/apache/gobblin/utils/HttpUtils.java

```diff
@@ -143,7 +143,7 @@ public static URI buildURI(String urlTemplate, Map<String, String> keys, Map<Str
    */
   public static void updateStatusType(ResponseStatus status, int statusCode, Set<String> errorCodeWhitelist) {
     if (statusCode >= 300 & statusCode < 500) {
-      List<String> whitelist = new ArrayList<>();
```

Comment:
We should try changing this name to "allow/denyList" as a part of this initiative for more inclusive language: https://iwww.corp.linkedin.com/wiki/cf/pages/viewpage.action?pageId=372446381

Reply:
We can have a new PR to include all of it throughout the codebase, good call out!